Description: Update Doxyfile as requested by doxygen
Author: Jonas Smedegaard <dr@jones.dk>
Last-Update: 2010-02-28
--- a/doc/Makefile.in
+++ b/doc/Makefile.in
@@ -25,6 +25,7 @@
 		echo "*** Sorry, can't build doc outside source dir"; exit 1; \
 	fi
 	sed 's/LIBSRTPVERSION/$(version)/' header.template > header.tex
+	doxygen -u
 	doxygen
 	sed 's/\subsection/\section/' latex/index.tex > latex/index.tmp
 	mv latex/index.tmp latex/index.tex
@@ -38,7 +39,7 @@
 	cp latex/refman.pdf crypto.pdf
 
 clean:
-	rm -rf latex/ header.tex
+	rm -rf latex/ header.tex Doxyfile.bak
 	for a in * ; do			                \
               if [ -f "$$a~" ] ; then rm -f $$a~; fi;	\
         done;
