From: Jonas Smedegaard <dr@jones.dk>
Date: Wed, 8 May 2019 16:16:16 -0400
Subject: Enable debugging in regression test shell script

Last-Update: 2011-05-25
---
 test/rtpw_test.sh | 5 ++++-
 1 file changed, 4 insertions(+), 1 deletion(-)

diff --git a/test/rtpw_test.sh b/test/rtpw_test.sh
index 238ec93..60cd134 100755
--- a/test/rtpw_test.sh
+++ b/test/rtpw_test.sh
@@ -37,6 +37,9 @@
 # OF THE POSSIBILITY OF SUCH DAMAGE.
 #
 
+set -e
+set -x
+
 case $(uname -s) in
     *CYGWIN*|*MINGW*)
         EXE=".exe"
@@ -60,7 +63,7 @@ ARGS="-b $key -a -e 128"
 # they are killed, those processes will linger.  Re-running the script
 # will get rid of them.
 
-killall rtpw 2>/dev/null
+killall -q rtpw || true
 
 if test -x $RTPW; then
 
